interchain-exchange/tutorial/04-order-pairs.md

```diff
@@ -94,7 +94,7 @@ When a packet sent with IBC is valid and received, it must be acknowledged.
 When the acknowledgement is successful, add the sell order book to the database.
```

Comment:
```suggestion
After the acknowledgement is successful, add the sell order book to the database.
```

Reply:
- when implies "right then" can we instead use "after" for the sequence of actions? 
(and hiya!)